docs/code/demos/advanced.py

```diff
@@ -40,18 +40,15 @@ def do_pipeline(arg1=Return('do_on_process'),
     Argument 'arg1' gets the return value of 'do_on_process'
     Argument 'arg2' gets the return value of function 'get_item'
     Argument 'arg3' is simply the value of a session parameter 'my_arg'"""
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).